FmpDevicePkg/FmpDevicePkg.dsc

```diff
@@ -55,14 +55,8 @@
   DebugPrintErrorLevelLib|MdePkg/Library/BaseDebugPrintErrorLevelLib/BaseDebugPrintErrorLevelLib.inf
   PcdLib|MdePkg/Library/BasePcdLibNull/BasePcdLibNull.inf
   VariablePolicyHelperLib|MdeModulePkg/Library/VariablePolicyHelperLib/VariablePolicyHelperLib.inf
-!ifdef CONTINUOUS_INTEGRATION

```

Comment:
It seems the only remaining place this is used in `FmpDevicePkg` is where it is set in FmpDevicePkg.ci.yaml. It seems useless there now and I suggest removing it with a `MU_CHANGE` comment.

Reply:
Fixed.